jormungandr/src/rest/v0/mod.rs

```diff
@@ -219,12 +219,18 @@ pub fn filter(
         .and_then(handlers::get_diagnostic)
         .boxed();
 
-    let committees = warp::path!("committees")
+    let committees = warp::path!("vote" / "active" / "committees")
         .and(warp::get())
         .and(with_context.clone())
         .and_then(handlers::get_committees)
         .boxed();
 
+    let vote_plans = warp::path!("vote" / "active" / "plans")
+        .and(warp::get())
+        .and(with_context.clone())
+        .and_then(handlers::get_active_vote_plans)
+        .boxed();
+
```

Comment:
Why not using `warp` grouping mechanism ? as done here https://github.com/input-output-hk/jormungandr/blob/7aef084df61a272b56c189322cb37c84aca70b38/jormungandr/src/rest/v0/mod.rs#L192-L208